mlrun/utils/notifications/notification/base.py

```diff
@@ -145,3 +146,13 @@ def _get_overview_type_and_url(
             overview_type = "Model endpoint"
 
         return overview_type, url
+
+    @staticmethod
+    def _load_json_from_str_param_with_default_json_value(
+        params, key, default_json_value
+    ):
+        if isinstance(params.get(key, ""), str):
```

Comment:
Can `params.get(key)` be a list or any other type besides a string?

Reply:
Yes, it can be, for example see validate_certs here:
`mail_notification = mlrun.model.Notification(kind="mail", when=["completed", "error"], name="ENXuIbyqEx", message="JMTpbLAldTZTAeWatuIpWzVTwSKdSaDq", condition="", severity="verbose", params={"start_tls": "true", "use_tls": "false", "validate_certs": False, "email_addresses": ["roei3000b@gmail.com","hegeg56369@inikale.com"]})`

Reply:
@roei3000b So you need to handle that case because right now you're only handling str